plugins/inputs/dpdk/dpdk.go

```diff
@@ -0,0 +1,240 @@
+// +build linux
+
+package dpdk
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/influxdata/telegraf/config"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	jsonparser "github.com/influxdata/telegraf/plugins/parsers/json"
+)
+
+const (
+	dpdkDescription  = "Reads metrics from DPDK applications using v2 telemetry interface."
+	dpdkSampleConfig = `
+  ## Path to DPDK telemetry socket. This shall point to v2 version of DPDK telemetry interface.
+  # socket_path = "/var/run/dpdk/rte/dpdk_telemetry.v2"
+
+  ## Duration that defines how long the connected socket client will wait for a response before terminating connection.
+  ## This includes both writing to and reading from socket.
+  ## Setting the value to 0 disables the timeout (not recommended)
+  # socket_access_timeout = "200ms"
```

Comment:
Is this a sensible default that works for "most users"?

Reply:
Comment added.